src/store/modules/product/actions.ts

```diff
@@ -37,7 +37,7 @@ const actions: ActionTree<ProductState, RootState> = {
       // Handled empty response in case of failed query
       if (resp.data) commit(types.PRODUCT_ADD_TO_CACHED_MULTIPLE, { products });
     } else {
-      console.error('Something went wrong')
+      console.log('Something went wrong')
```

Comment:
When an error we use `error` for logging, also this console is required for debugging purposes, please revert.